test/common/upstream/health_checker_impl_test.cc

```diff
@@ -659,8 +659,10 @@ TEST_F(HttpHealthCheckerImplTest, Success) {
   EXPECT_CALL(*test_sessions_[0]->timeout_timer_, enableTimer(_, _));
   health_checker_->start();
 
-  EXPECT_CALL(runtime_.snapshot_, getInteger("health_check.max_interval", _));
-  EXPECT_CALL(runtime_.snapshot_, getInteger("health_check.min_interval", _))
+  EXPECT_CALL(runtime_.snapshot_,
+              getInteger("health_check.max_interval", testing::Matcher<uint64_t>(_)));
```

Comment:
Prefer different name for `getInteger` which will revert these changes.

Reply:
Noted and done on next push